server/integration_tests/explore_test.py

```diff
@@ -111,6 +111,19 @@ def handle_response(self,
                       check_detection=False,
                       detector=None):
     dbg = resp['debug']
+
+    # sort variables in the response because variable scores can change between
+    # runs. Sort by scores cut off after 6 digits after the decimal and for
+    # variables with the same truncated score, sort alphabetically
+    resp_var_to_score = {}
+    for i, sv in enumerate(dbg['sv_matching']['SV']):
+      score = dbg['sv_matching']['CosineScore'][i]
+      resp_var_to_score[sv] = float("{:.6f}".format(score))
+    if 'variables' in resp:
+      sorted_variables = sorted(resp['variables'],
+                                key=lambda x: (-resp_var_to_score[x], x))
+      resp['variables'] = sorted_variables
```

Comment:
should all the golden scores change to 6 digit now?

Reply:
no, this just updates the ordering of variables